routes/index.js

```diff
@@ -0,0 +1,10 @@
+const router = require('express').Router()
+const userRoutes = require('./userRoutes')
+
+router.get('/', (req, res) => {
+  res.json({"status": "terkoneksi"})
+})
+
+router.use('/', userRoutes)
```

Comment:
Di user routes kan depannya ada `/api` semua kan 
kenapa disini ga dibuat

```suggestion
router.use('/api', userRoutes)
```

jadi di userRoutes nya udah ga usah pake `/api/` lagi.